frontend/src/pages/modelServing/screens/global/InferenceServiceListView.tsx

```diff
@@ -26,8 +25,6 @@ const InferenceServiceListView: React.FC<InferenceServiceListViewProps> = ({
   } = React.useContext(ModelServingContext);
   const [searchType, setSearchType] = React.useState<SearchType>(SearchType.NAME);
   const [search, setSearch] = React.useState('');
-  const [page, setPage] = React.useState(1);
-  const [pageSize, setPageSize] = React.useState(MIN_PAGE_SIZE);
   const sortInferenceService = useTableColumnSort<InferenceServiceKind>(
```

Comment:
I think you want to remove this too -- no? `.transformData` is the sort transform -- which is done by your Table.